<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryForm</class>
 <widget class="QWidget" name="MemoryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,8,1">
   <item>
    <widget class="WordDefsForm" name="wgtDefsForm" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="wgtPics" native="true"/>
   </item>
   <item>
    <widget class="MemoryControlForm" name="wgtControl" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WordDefsForm</class>
   <extends>QWidget</extends>
   <header>WordDefsForm.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MemoryControlForm</class>
   <extends>QWidget</extends>
   <header>MemoryControlForm.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
